File: PyUtils/utils/JsonUtil.py
# -*- coding: utf-8 -*-

# Json 工具类

import json

__author__ = 'qiunet'


def toJsonString(obj: object) -> str:
    return json.dumps(obj, default=lambda obj: obj.__dict__)


def getGeneralObject(jsonString: str):
    return json.loads(jsonString)

File: Excel2Cfgs/src/main/java/org/qiunet/excel2cfgs/common/utils/JsonUtil.java
package org.qiunet.excel2cfgs.common.utils;

import com.alibaba.fastjson.JSON;

/***
 *
 * @author qiunet
 * 2021/11/15 14:01
 */
public class JsonUtil {

	public static String toJsonString(Object obj) {
		return JSON.toJSONString(obj);
	}

	public static <T> T getGeneralObject(String json ,Class<T> c){
		return JSON.parseObject(json, c);
	}
}
